### ISSUE TITLE
Annotating optional regions?

### ISSUE THREAD

**Issue by brollb** [symbench/scm#363]
Maybe these should just be treated as a note?
![DeepinScreenshot_20211111104536](https://user-images.githubusercontent.com/4982789/141335966-221c6eea-ecc1-4ae2-aa63-bcac202998a3.png)

**Comment by brollb** [comment]
For now, I just labeled it as "Abstract" with the name "Optional"

**Comment by umesh-timalsina** [comment]
The component looks like a relay/ switch.

**Comment by brollb** [comment]
Yeah, though I was focusing on the dotted box that was designating it as optional here since it isn't clear how that should be annotated.
